parachain/src/wasm_executor/mod.rs

```diff
@@ -172,7 +192,12 @@ pub fn validate_candidate(
 ) -> Result<ValidationResult, ValidationError> {
 	match isolation_strategy {
 		IsolationStrategy::InProcess => {
-			validate_candidate_internal(validation_code, &params.encode(), spawner)
+			validate_candidate_internal(
+				&ExecutorCache::default(),
```

Comment:
If we create a new default `ExecutorCache` here, how is that different from calling `WasmExecutor::new` within `validate_candidate_internal`? AFAICT the only difference is that we're creating the struct one frame deeper in the stack.

Reply:
Did you read the description of the pr? It is about a cache for the external process and not for when being in process.

Reply:
Yes, but missed that. Makes more sense now.